recipes/spdlog/all/conanfile.py

```diff
@@ -45,7 +45,10 @@ def configure(self):
             raise ConanInvalidConfiguration("wchar is not yet supported under windows")
 
     def requirements(self):
-        self.requires("fmt/6.0.0")
+        if self.version == "1.5.0":
+            self.requires("fmt/6.1.2")
```

Comment:
fmt/6.1.2 does not it work with older spdlog versions?

Reply:
Yes you are right, I misread the changelog. I will change it.